app/packages/core/src/plugins/SchemaIO/components/TableView.tsx

```diff
@@ -170,13 +182,14 @@ export default function TableView(props: ViewPropsType) {
                         selectedCells.has(coordinate) ||
                         isRowSelected ||
                         selectedColumns.has(columnIndex);
-                      return (
+                      
+                      const tooltip = tooltipMap[coordinate]; // Check if there's a tooltip for the cell
```

Comment:
Here we don't check if coordinate is a valid key, the assumption is that tooltipMap will return null if the key is not in the map